test/scale-infra/README.md

```diff
@@ -0,0 +1,69 @@
+A scalable Kubernetes-based testbed for KubeStellar Performance Tests
+---------------------------------------------------------------------
+
+<img src="images/ks-scale-test-infra.png" width="60%" height="60%" title="ks-scale-test-infra">
+
+### Requirements
+AWS EC2 is the environment used to setup the testbed. So AWS credentials are required.
+They can be set by environment variables, e.g. `AWS_ACCESS_KEY` and `AWS_SECRET_KEY`.
+
+### Overview
+
+The testbed consists of (from bottom to top):
+- Networking and stroage resources from AWS;
+- AWS EC2 instances;
+- Kubernetes cluster(s);
+- Workload(s): Kubestellar Core and WEC components 
+
+
+### Deployment Instructions
+
+1. Deploy the core region: 
+
+```
+./deploy_core_region.sh --region us-east-2  --num_workers 2 --instance_type t2.xlarge   --aws_key_name mykey  --archt x86_64
+```
+
+The above command creates the required AWS infrastructure including a security group and EC2 instances on AWS.
+EC2 region, numbers of master nodes and worker nodes, and instance type can be specified in playbook `vars`. Then, it creates a Kubernetes cluster deployed using Kubeadm and KubeStellar core components will be installed.
+
+
+2. Create the WEC hosting instances:
+
+```
+./deploy_edge_region.sh --region us-east-2 --num_workers 1 --instance_type t2.2xlarge --aws_key_name  mykey  --archt x86_64
```

Comment:
At this point the reader is wondering "what is a worker?"

Reply:
Agree! Replaced with `num_hosts` instead.